src/services/ApiCalls.tsx

```diff
@@ -0,0 +1,63 @@
+import { URL_SERVER } from "constants/serverConnexion";
+
+const calculateAge = (birthday: any) => {
+  var ageDifMs = Date.now() - birthday;
+  var ageDate = new Date(ageDifMs); // miliseconds from epoch
+  return Math.abs(ageDate.getUTCFullYear() - 1970);
```

Comment:
Is the abs really needed here?

Reply:
this looks weird indeed. I'm sure there is a way to do this without having to hard-code "1970"